src/commands/google.rs

```diff
@@ -1 +1,26 @@
+use super::is_in_incorrect_channel;
+use serenity::{
+    client::Context,
+    framework::standard::{macros::command, CommandResult},
+    model::channel::Message,
+};
 
+#[command]
+#[aliases("browser", "google")]
+#[description("Teaches users how to paste code neatly")]
+#[usage("")]
+#[example = ""]
+#[help_available]
+#[bucket = "potentially_big_output"]
+pub(crate) async fn codeblock(ctx: &Context, msg: &Message) -> CommandResult {
```

Comment:
you probably want to look at the codeblock's command. As that one has a bigger delay and can run from any channel

Reply:
done.

Reply:
I think you need to push,as I don't see the changes yet